src/vs/platform/quickinput/browser/quickInputController.ts

```diff
@@ -546,7 +546,9 @@ export class QuickInputController extends Disposable {
 			input.ignoreFocusOut = !!options.ignoreFocusLost;
 			input.matchOnDescription = !!options.matchOnDescription;
 			input.matchOnDetail = !!options.matchOnDetail;
-			input.sortByLabel = !!options.sortByLabel;
+			if (options.sortByLabel) {
```

Comment:
Shouldn't this be `typeof options.sortByLabel === 'boolean'` so that I can disable sorting by label? Isn't otherwise sorting by label the default?